src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,14 @@ public function getConfigTreeBuilder()
         $rootNode = $treeBuilder->root('web_profiler');
 
         $rootNode
+            ->beforeNormalization()
+                ->ifTrue(function ($v) { return isset($v['position']); })
```

Comment:
should be ``array_key_exists`` IMO, to account for ``null``. They should not configure it at all